src/iomusxml.cpp

```diff
@@ -576,6 +576,7 @@ bool MusicXmlInput::ReadMusicXml(pugi::xml_node root)
         if (!measure) {
             LogWarning("MusicXML import: Element '%s' could not be added to measure '%s'",
                 iter->second->GetClassName().c_str(), iter->first.c_str());
+            m_controlElements.erase(iter);
```

Comment:
This screws up the iterator and does not fix the memory leak because the ControlElement is still not deleted.

Reply:
Ok, so we just clear all remaining elements.